### PR TITLE
Docs: copyedit iOS Card pages

### PR DESCRIPTION
**Pull request by dangerismycat**

### CODE REVIEW

File: docs/markdown/ios/card/card.header.md [hunk 1]
```diff
@@ -1,41 +1,28 @@
...
 
-<ImgContainer src="https://i.pinimg.com/originals/11/90/6b/11906b1575ba50e08d48b8a0bf20e805.png" alt="Example of a Card.Header with a highlighted area to show how it is placed and arranged in a Card." noPadding color="background-elevation-accent"/>
-
-
-## Usage guidelines
-
-<TwoCol>
-<Group>
-<Do title="When to use"/>
-Show a navigational arrow or more menu within the Card.Header to direct the user to the appropriate additional content.
```
Comment (dangerismycat) [inline docs/markdown/ios/card/card.header.md:15]: These two points aren't "when to use"/"when not to use", they're "Do"/"Don't"'s. Not sure if there are better usage guidelines to add here

File: docs/markdown/ios/card/card.pins.md [hunk 2]
```diff
@@ -1,11 +1,10 @@
 ---
 title: Card.Pins
-description: The Boards preview block utilizes two of the Board and Section components to present a collection of imagery that typically represents Card.Boards.
+description: Card.Pins is a preview block utilizing two of the Pin and Section components to present a collection of imagery.
```
Comment (dangerismycat) [inline docs/markdown/ios/card/card.pins.md:3]: The previous copy here was referencing Card.Boards, so seemed to be incorrect. Not sure if this new copy is factually correct

File: docs/markdown/ios/card/card.pins.md [hunk 3]
```diff
@@ -55,10 +56,13 @@ People use Apple’s accessibility features, such as reduced transparency, Voice
 *1. Image one / 2. Image two / 3. Image three*
 
 ## Variants
-The Card.Boards preview block utilizes the [Gestalt Board and Section Figma component](http://pinch.pinadmin.com/section-and-board-component-figma-iOS). The Card.Boards preview block accepts all of Board and Sections variants except for CTAs. Optional elements include: lock icon, plus icon, pretext, Card.Boards name, collaborators and subtext.
+
+Card.Pins is a preview block utilizing two of the Pin and Section components to present a collection of imagery. The Card.Pins preview block accepts all of Pin and Sections variants except for CTAs. Optional elements include: lock icon, plus icon, pretext, Card.Pins name, collaborators and subtext.
```
Comment (dangerismycat) [inline docs/markdown/ios/card/card.pins.md:60]: The previous copy here was referencing Card.Boards, so seemed to be incorrect. Not sure if this new copy is factually correct

File: docs/markdown/ios/card/card.header.md (hunk 1 above)
Comment (hectoid) [reply docs/markdown/ios/card/card.header.md:15]: good catch! these are wrong. Should be: 
https://coda.io/d/_dqIC9SDPsU9/CardHeader-subcomponent_supz_#_luOB4

Then best practices are: https://coda.io/d/_dqIC9SDPsU9/CardHeader-subcomponent_supz_#_luPnt

File: docs/markdown/ios/card/card.pins.md (hunk 3 above)
Comment (hectoid) [reply docs/markdown/ios/card/card.pins.md:60]: sorry, another cut and paste error. It's probably good to have a Variants intro paragraph, but the original coda doc doesn't have one. The embeds are also incorrect. https://coda.io/d/_dqIC9SDPsU9/Pins-Preview-Block_suJtg#_lukRk

File: docs/markdown/ios/card/card.pins.md [hunk 4]
```diff
@@ -1,29 +1,107 @@
...
+
+### Anatomy
+
+<iframe style={{border:0}} width="100%" height="450" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A76473%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1" allowFullScreen></iframe>
+
+### Variants
+
+#### 2-up
+
+<iframe style={{border:0}} width="100%" height="312" src="https://www.figma.com/file/AHcKJDgb7E7YswlgW1wY8E/Gestalt-for-iOS?type=design&node-id=19826%3A75261&mode=design&t=IclHzfbakvSzsqp8-1" allowFullScreen></iframe>
```
Comment (hectoid) [inline docs/markdown/ios/card/card.pins.md:65]: I'm getting a failure to connect message here. Not sure if it's Figma or a wrong URL. Here is a URL for the embed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19826%253A75261%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1

Also, height="360"

File: docs/markdown/ios/card/card.pins.md [hunk 5]
```diff
@@ -1,29 +1,107 @@
...
+
+#### 3-up (default)
+
+<iframe style={{border:0}} width="100%" height="312" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A76866%26mode%3Ddesign%26t%3DIclHzfbakvSzsqp8-1" allowFullScreen></iframe>
+
+#### 4-up
```
Comment (hectoid) [inline docs/markdown/ios/card/card.pins.md:71]: Sorry, wrong embed link. use this one: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19826%253A75351%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1

height: 360

**Review by hectoid — CHANGES_REQUESTED** [review]

File: docs/markdown/ios/card/card.pins.md [hunk 6]
```diff
@@ -39,26 +117,22 @@ fullwidth: true
...
 
 [Accessible design on iOS](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/introduction/)
 [Accessible development on iOS](https://developer.apple.com/accessibility/ios/)
 
-## Design tokens
-<iframe style={{border:0}} width="100%" height="178" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A76699%26t%3DaliDwdC0C3b2VkAb-1" allowFullScreen></iframe>
+### Design tokens
 
-## Anatomy
+<iframe style={{border:0}} width="100%" height="624" src="https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19801%253A75095%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1" allowFullScreen></iframe>
```
Comment (hectoid) [inline docs/markdown/ios/card/card.pins.md:129]: ugh, sorry, one last one. updated embed link: "https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FAHcKJDgb7E7YswlgW1wY8E%2FGestalt-for-iOS%3Ftype%3Ddesign%26node-id%3D19800%253A79954%26mode%3Ddesign%26t%3DmAYek1WBSRRnRvq3-1

**Review by hectoid — APPROVED** [review]
the embed for anatomy is correct, so it's just taking time to update I guess. thanks for all of the changes!

**Pull request merged**